lib/tldr-lint.js

```diff
@@ -180,3 +183,19 @@ linter.process = function(page, verbose, alsoFormat) {
 
   return result;
 };
+
+linter.processFile = function(file, verbose, alsoFormat) {
+  var result = linter.process(fs.readFileSync(file, 'utf8'), verbose, alsoFormat);
+  var filename = path.basename(file);
+  if (filename.substring(filename.length - 3) !== '.md') {
```

Comment:
I was just wondering if it was worth using `path.extname(file) !== '.md'` here for easier readability. But not sure if it's really necessary.

Reply:
Good point, changed. Slipped my mind to use the `path` functions for this stuff.